targets/psa_switch/psa_switch.cpp

```diff
@@ -342,6 +341,21 @@ PsaSwitch::ingress_thread() {
       continue;
     }
 
+    // resubmit - these packets get immediately resub'd to ingress, and skip
+    //            deparsing for now, do not move below multicast or deparse
```

Comment:
what does "for now" mean here? resubmitted packets should not be deparsed so I don't think this should change in the future.

Reply:
I meant it as "if I'm marked as resubmit, skip deparsing on this pass into ingress, but I'll get deparsed in a later pass" - but perhaps my wording is off.

Reply:
also I guess what I said isn't true for dropped packets in our implementation of bmv2... I'll just remove the "for now" from this comment